benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -80,46 +80,47 @@ msgstr ""
 "benefit to the rider's contactless card of choice."
 
 msgid "core.pages.help.payment_options"
-msgstr "Payment options"
+msgstr "What kind of credit card can I use?"
 
-msgid "core.pages.help.payment_options.p[0]"
+msgid "eligibility.pages.start.modal.p[0]"
 msgstr ""
-"In order to activate your discount, you need a bank card that has the "
-"contactless symbol on it. The contactless symbol is four curved lines that "
-"looks like this:"
+"Benefits only works with contactless debit or credit cards. We do not "
+"currently support mobile wallets or digital cards, such as Apple Pay, Google "
+"Wallet, or Samsung Wallet."
+
+msgid "eligibility.pages.start.modal.p[1]"
+msgstr "The contactless symbol is four curved lines, like this:"
 
 msgctxt "image alt text"
 msgid "core.icons.contactless"
 msgstr "Four curved lines on contactless-enabled cards"
 
-msgid "core.pages.help.payment_options.p[1]"
-msgstr ""
-"A contactless bank card can be a credit card or a debit card and must "
-"include a Visa or Mastercard logo."
+msgid "eligibility.pages.start.modal.p[2]"
+msgstr "Your card must include a Visa or Mastercard logo."
 
-#, python-format
-msgid "core.pages.help.payment_options.p[2][0]%(website)s"
+msgid "eligibility.pages.start.modal.p[3]"
 msgstr ""
-"Don’t have access to a bank card? You can request a contactless card from "
-"one of the companies that offer free contactless prepaid debit cards, such "
-"as the <a href=\"%(website)s\" target=\"_blank\" rel=\"noopener noreferrer"
-"\">Cash App Visa Debit Card</a> "
+"Don’t have access to a contactless card? You can request a contactless card "
+"from your bank, or one of the companies that offer free contactless prepaid "
+"debit cards, such as the Cash App Visa debit card or the Venmo Mastercard "
+"debit card."
 
 #, python-format
-msgid "core.pages.help.payment_options.p[2][1]%(website)s"
+msgid "eligibility.pages.start.modal.p[4.1]%(agency_short_name)s"
 msgstr ""
-"or the <a href=\"%(website)s\" target=\"_blank\" rel=\"noopener noreferrer"
-"\">Venmo Mastercard Debit Card</a>."
+"You can still get your transit benefit by going through your "
+"%(agency_short_name)s application process. For updates on additional "
+"options, please check back on this website, or contact "
 
-msgid "core.pages.help.payment_options.p[3]"
+#, python-format
+msgid "eligibility.pages.start.modal.p[4.2]%(agency_info_url)s"
 msgstr ""
-"You can still get your discount by going through your local transit "
-"provider’s application process."
+"<a href=\"%(agency_info_url)s\" target=\"_blank\" rel=\"noopener noreferrer"
+"\">"
 
-msgid "core.pages.help.payment_options.p[4]"
-msgstr ""
-"For updates on additional options, please check back on this website, or get "
-"in touch with your local transit provider."
+#, python-format
+msgid "eligibility.pages.start.modal.p[4.3]%(agency_short_name)s"
+msgstr "%(agency_short_name)s</a>."
```

Comment:
Part of my question above, do we need this in the PO files?

Reply:
See https://github.com/cal-itp/benefits/pull/1558#discussion_r1272757107.

I don't think we need this split into multiple pieces in the PO file, probably just one right?

Reply:
Yeah, I had it as one in [the first commit](https://github.com/cal-itp/benefits/pull/1558/commits/c5b15bab6de198fb5d7d150d138d24fd113a932c#diff-8b77e09357e0ac2a23a88239a579cb8218a1c212a5d816af33125cf3690e7a16R108-R114) of this PR. In the second commit of this PR, I refactored the template to use the same msgids that the Contactless Card modal is using, as implemented in #1523.

I can update both the Help page and the modal to use one entry if that's the direction we'd like to go.

Reply:
I see. Does the modal need it to be split up? I think we can assume one long sentence with the same `short_name` replacement used a few different times and the link within -- does that make sense?

Reply:
No, I don't think the modal needs it split up. I made it one entry in https://github.com/cal-itp/benefits/pull/1558/commits/e9849f5da47092c681dce51ddeac1ca5d9a80da9 and also changed the `msgstr` to use placeholders like we're doing everywhere else until we have the real copy.

Reply:
Ah, I think for this one I was hoping we could just finish it out -- we already have the `short_name` available in the context along with the `info_url`. The translation shouldn't change the agency's short name that we display.

Reply:
Oh ok, got it. I've made that change in [9159af5](https://github.com/cal-itp/benefits/pull/1558/commits/9159af53560f46d6688df64a94eaf07ec931453a)